sklearn/metrics/_plot/base.py

```diff
@@ -82,7 +82,7 @@ def _get_response(X, estimator, response_method, pos_label=None):
         The class considered as the positive class when computing
         the metrics.
     """
-    classification_error = "{} should be a binary classifier".format(
+    classification_error = "Expected a binary classifier, but got {}".format(
```

Comment:
Could you use a f-string here, it's less verbose and helps readability (ref: https://github.com/scikit-learn/scikit-learn/issues/17327)

Reply:
Thank you , Lucy